src/components/calculatorUi.js

```diff
@@ -0,0 +1,77 @@
+import React,{useState} from 'react';
+import Button from './button';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const btnFirst =['AC','+/-','%','/','7','8','9','*','4','5','6','-','1','2','3','+','0','.','=']
+  const [inputValue, setInputValue]=useState('')
+  const [numberOne,setNumberOne]=useState('')
+  const [numberTwo,setNumberTwo]=useState(0)
+  const [operator,setOperator]=useState('')
+
+
+  const handleNumber=(e)=>{
+    const value = e.target.textContent
+
+    if(numberOne !== ''){
+      if(value === '%' || value === '-' || value === '*' || value === '+' || value === '/' ){
+        console.log(numberOne)
+        setInputValue('')
+        setOperator(()=>value)
+        setInputValue(`${value}`)
+        return
```

Comment:
- [ ] I noticed that when I click any operation button, the subsequent numbers do not string along. They are entered as a single digit. This affects the calculations and produces the wrong results.

![chrome-capture-2023-1-14](https://user-images.githubusercontent.com/77106962/218843259-403d8db5-3494-4432-b2ed-3e862aef23d3.gif)

![chrome-capture-2023-1-14 (1)](https://user-images.githubusercontent.com/77106962/218844426-758bdd71-8ce2-46d9-9372-faf9ce2ecbb7.gif)

Reply:
The code works fine on my local machine and it produces the expected the result too